modules/migration-creating-migration-plan-cam.adoc

```diff
@@ -9,40 +9,22 @@
 
 You can create a migration plan in the {mtc-full} ({mtc-short}) web console.
 
-You can use _direct image migration_ and _direct volume migration_ to migrate images or volumes directly from the source cluster to the target cluster. Direct migration improves performance significantly.
-
 .Prerequisites
 
-* You must add source and target clusters and a replication repository to the {mtc-short} web console.
-* Clusters:
-** The clusters must have network access to each other.
-** The clusters must have network access to the replication repository.
-** The clusters must be able to communicate using OpenShift routes on port 443.
-** The clusters must have no `Critical` conditions.
-** The clusters must be in a `Ready` state.
-
-* The migration plan name must not exceed 253 lower-case alphanumeric characters (`a-z, 0-9`) and must not contain spaces or underscores (`_`).
-* PV `Move` copy method: The clusters must have network access to the remote volume.
-* PV `Snapshot` copy method:
-** The clusters must have the same cloud provider (AWS, GCP, or Azure).
-** The clusters must be located in the same geographic region.
-** The storage class must be the same on the source and target clusters.
-
-* Direct image migration:
-** The source cluster must have its internal registry exposed to external traffic.
-** The exposed registry route of the source cluster must be added to the cluster configuration by using either the {mtc-short} web console or the `exposedRegistryPath` parameter in the `MigCluster` CR manifest.
-
-* Direct volume migration:
-** The PVs to be migrated must be valid.
-** The PVs must be in a `bound` state.
-** The PVs must be copied by using the file system copy method.
-** You must configure a TCP proxy for Stunnel for direct volume migration in a proxy environment.
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* You must ensure that the same {mtc-short} version is installed on all clusters.
+* You must add the clusters and the replication repository to the {mtc-short} web console.
```

Comment:
Should these two be past tense as the steps above show me how to do this? 
_You must ensure that the same {mtc-short} version was installed_
_You must have added_

Reply:
No, because we do not assume that a module is always used in the same context. Therefore, there is no reason to change the tense.